web/src/app/WebApp/Installers/MediaWiki/MediaWikiSetup.php

```diff
@@ -53,10 +53,10 @@ public function install(array $options = null)
 
         $sslEnabled = ($status->json[$this->domain]['SSL'] == 'no' ? 0 : 1);
 
-        $webDomain = ($sslEnabled ? "https://" : "http://") . $this->domain . "/";
```

Comment:
I've removed this "/", so URL can be https://domain.tld/, instead of https://domain.tld//.